ktorm-core/src/main/kotlin/me/liuwj/ktorm/expression/SqlFormatter.kt

```diff
@@ -73,7 +73,7 @@ open class SqlFormatter(
         }
     }
 
-    protected val String.quoted: String get() {
+    open protected val String.quoted: String get() {
         if (this.toUpperCase() in database.keywords || !this.isIdentifier) {
```

Comment:
I'm not sure what `isIdentifier` signifies - while I don't believe there are any cases where quoting the column name in PostgreSQL will result in invalid syntax, there may be an edge case of which I'm unaware.